changelog.json

```
@@ -1,5 +1,53 @@
 {
   "releases": {
+    "1.7.1": [
+      "[New] You can now choose to bring your changes with you to a new branch or stash them on the current branch when switching branches - #6107",
+      "[New] Rebase your current branch onto another branch using a guided flow - #5953",
+      "[New] Recently opened repositories appear at the top of the repository list - #7132",
```

Didn't see https://github.com/desktop/desktop/pull/7505 in here but I might have missed it. Either way I propose we put this up at the new section.

```suggestion
      "[New] Recently opened repositories appear at the top of the repository list - #7132",
      "[New] Suggested next steps now includes suggestion to create a pull request after publishing a branch - #7505",
```